lib/rspec/core/formatters/message_formatter.rb

```diff
@@ -0,0 +1,28 @@
+module RSpec
+  module Core
+    module Formatters
+      # @api private
+      # Formatter for providing message output as a fallback when no other
+      # profiler implements #message
+      class MessageFormatter
```

Comment:
> Since this isn't generally needed, and is only used as a fallback, WDYT of the name FallbackMessageFormatter?

I'd prefer `MessageFormatter` as theres nothing specific to being a fallback but I don't feel that strongly about it.

> Also, would it be worthwhile to subclass BaseFormatter so we get all the output.sync logic for this formatter as well?

`BaseFormatter` adds other things I don't feel belong in here, prehaps we should move the sync stuff to the `FormatterLoader` or something else?